MainWindow.xaml.cs

```diff
@@ -24,7 +24,7 @@ public partial class MainWindow : Window
         // Private UI-related variables
         private List<Tuple<char, string, bool>> _drives { get; set; }
         private List<int> _driveSpeeds { get { return new List<int> { 1, 2, 3, 4, 6, 8, 12, 16, 20, 24, 32, 40, 44, 48, 52, 56, 72 }; } }
-        private List<Tuple<string, KnownSystem?, DiscType?>> _systems { get; set; }
+        private List<Tuple<string, KnownSystem?>> _systems { get; set; }
```

Comment:
For the time being, can you also have a persistent _discTypes list here as well? That helps keep track of what is being used globally.

Reply:
Sure, I'll build a List<DiscType?> of all currently used types for the systems, which sounds like what we're looking for since it's just to track it.